gitee/payload.go

```diff
@@ -0,0 +1,289 @@
+}
+
+type IssueEventPayload struct {
+	Action      *string         `json:"action,omitempty"`
+	Issue       *IssueHook      `json:"issue,omitempty"`
+	Repository  *ProjectHook    `json:"repository,omitempty"`
+	Project     *ProjectHook    `json:"project,omitempty"`
+	Sender      *UserHook       `json:"sender,omitempty"`
+	TargetUser  *UserHook       `json:"target_user,omitempty"`
+	User        *UserHook       `json:"user,omitempty"`
+	Assignee    *UserHook       `json:"assignee,omitempty"`
+	UpdatedBy   *UserHook       `json:"updated_by,omitempty"`
+	IID         string          `json:"iid,omitempty"`
+	Title       *string         `json:"title,omitempty"`
+	Description *string         `json:"description,omitempty"`
+	State       *string         `json:"state,omitempty"`
+	Milestone   *string         `json:"milestone,omitempty"`
+	URL         *string         `json:"url,omitempty"`
+	Enterprise  *EnterpriseHook `json:"enterprise,omitempty"`
+	HookName    *string         `json:"hook_name,omitempty"`
+	Password    *string         `json:"password,omitempty"`
+}
+
+type MergeRequestEventPayload struct {
+	Action         *string          `json:"action,omitempty"`
+	ActionDesc     *string          `json:"action_desc,omitempty"`
+	PullRequest    *PullRequestHook `json:"pull_request,omitempty"`
+	Number         int64            `json:"number,omitempty"`
+	IID            int64            `json:"iid,omitempty"`
+	Title          *string          `json:"title,omitempty"`
+	Body           *string          `json:"body,omitempty"`
+	State          *string          `json:"state,omitempty"`
+	MergeStatus    *string          `json:"merge_status,omitempty"`
+	MergeCommitSha *string          `json:"merge_commit_sha,omitempty"`
+	URL            *string          `json:"url,omitempty"`
+	SourceBranch   *string          `json:"source_branch,omitempty"`
+	SourceRepo     *RepoInfo        `json:"source_repo,omitempty"`
+	TargetBranch   *string          `json:"target_branch,omitempty"`
+	TargetRepo     *RepoInfo        `json:"target_repo,omitempty"`
+	Project        *ProjectHook     `json:"project,omitempty"`
+	Repository     *ProjectHook     `json:"repository,omitempty"`
+	Author         *UserHook        `json:"author,omitempty"`
+	UpdatedBy      *UserHook        `json:"updated_by,omitempty"`
+	Sender         *UserHook        `json:"sender,omitempty"`
+	TargetUser     *UserHook        `json:"target_user,omitempty"`
+	Enterprise     *EnterpriseHook  `json:"enterprise,omitempty"`
+	HookName       *string          `json:"hook_name,omitempty"`
+	Password       *string          `json:"password,omitempty"`
+}
+
+type TagEventPayload struct {
+	Action *string `json:"action,omitempty"`
+}
+
+// RepoInfo : Repository information
+type RepoInfo struct {
+	Project    *ProjectHook `json:"project,omitempty"`
+	Repository *ProjectHook `json:"repository,omitempty"`
+}
+
+// LabelHook : Label, issue and pull request labels
+type LabelHook struct {
+	Id    int32  `json:"id,omitempty"`
```

Comment:
Interesting thing! Sometimes you use int32, but sometimes int64. Does it make sense to have different types for IDs?